packages/remix-docs-gen/src/export.ts

```diff
@@ -1,20 +1,55 @@
 import chalk from "chalk";
 import fs from "fs-extra";
 import * as path from "path";
-import { ComputedRoute } from "./types";
+import { ComputedRoute, Route, RouteDocsAggregate } from "./types";
 import { logSuccess } from "./utils";
 
+const generateDocsForRoutes = (routes: ComputedRoute[]) => {
```

Comment:
here I just abstracted your map / join into a function so I can call it to generate multiple types